flow360/examples/om6wing/flow360.json

```diff
@@ -1,90 +1,104 @@
 {
-    "geometry" : {
-        "refArea" : 1.15315084119231,
-        "momentCenter" : [0.0, 0.0, 0.0],
-        "momentLength" : [1.47602, 0.801672958512342, 1.47602]
-    },
-    "volumeOutput" : {
-        "outputFormat" : "tecplot",
-        "animationFrequency" : -1,
-        "primitiveVars" : true,
-        "vorticity" : false,
-        "residualNavierStokes" : false,
-        "residualTurbulence" : false,
-        "T" : false,
-        "s" : false,
-        "Cp" : false,
-        "mut" : false,
-        "mutRatio" : false,
-        "Mach" : true
-    },
-    "surfaceOutput" : {
-        "outputFormat" : "tecplot",
-        "animationFrequency" : -1,
-        "primitiveVars" : true,
-        "Cp" : true,
-        "Cf" : true,
-        "CfVec" : false,
-        "yPlus" : false,
-        "wallDistance" : false
-    },
-
-    "sliceOutput": {
-        "animationFrequency": -1,
-        "animationFrequencyOffset": 0,
-        "coarsenIterations": 0,
-        "outputFormat": "tecplot",
-        "outputFields": [ "Cp" ],
-        "slices": {
-            "x0": {
-                "sliceNormal": [ 1, 0, 0 ],
-                "sliceOrigin": [ 0, 0, 0 ],
-                "outputFields": [ "Mach" ]
-            },
-            "y1": {
-                "sliceNormal": [ 0, 1, 0 ],
-                "sliceOrigin": [ 2, 1, 0 ],
-                "outputFields": [ "T" ]
-            }
+    "unitSystem": {
+        "mass": {
+            "value": 1.0,
+            "units": "kg"
+        },
+        "length": {
+            "value": 1.0,
+            "units": "m"
+        },
+        "time": {
+            "value": 1.0,
+            "units": "s"
+        },
+        "temperature": {
+            "value": 1.0,
+            "units": "K"
+        },
+        "velocity": {
+            "value": 1.0,
+            "units": "m/s"
+        },
+        "area": {
+            "value": 1.0,
+            "units": "m**2"
+        },
+        "force": {
+            "value": 1.0,
+            "units": "N"
+        },
+        "pressure": {
+            "value": 1.0,
+            "units": "Pa"
+        },
+        "density": {
+            "value": 1.0,
+            "units": "kg/m**3"
+        },
+        "viscosity": {
+            "value": 1.0,
+            "units": "Pa*s"
+        },
+        "angular_velocity": {
+            "value": 1.0,
+            "units": "rad/s"
```

Comment:
I think we can serialise it to SI

Reply:
Yes, hasn't been updated yet since i think there was no comparison tests for it.